hilti/include/rt/types/vector.h

```diff
@@ -136,14 +136,21 @@ class Vector : public std::vector<T, Allocator>, public hilti::rt::detail::itera
     Vector& operator=(const Vector&) = default;
     Vector& operator=(Vector&&) noexcept = default;
 
-    const T& operator[](uint64_t i) const {
+    const T& operator[](uint64_t i) const& {
```

Comment:
For my education, what's the difference between `const` and `const&` here?

Reply:
Oh, the answer is probably in my next note below.

Reply:
Exactly.